data/variant-mappings.json

```diff
@@ -0,0 +1,8 @@
+{
+    "localhost:3001": {
+        "courses": {
+            "1": "main",
```

Comment:
@mjoss17 - Sorry, should have caught this the first time, but let's remove this mapping for course 1 so the test case validates the sparse mapping definitions we expect in practice.